backend/src/models/EntrepriseManager.js

```diff
@@ -25,6 +25,27 @@ class EntrepriseManager extends AbstractManager {
       ]
     );
   }
+
+  update(entreprise) {
+    return this.connection.query(
```

Comment:
update ${this.table} set ? where id = ?, [entreprise, entreprise.id]... Si tu ajoute un champs, tu n'as pas besoin de modifier ta requête